src/api/auth/util.rs

```diff
@@ -1,34 +1,12 @@
 use crate::api::RedisConn;
 use crate::models::*;
-use crate::schema::user::{self, otps_sent};
+use crate::schema::user::{self};
 use crate::util::function;
 use crate::{constants::INITIAL_RATING, error::DieselError};
 use anyhow::Result;
 use diesel::prelude::*;
-use pwhash::bcrypt;
 use rand::{distributions::Alphanumeric, Rng};
 use redis::Commands;
-use std::time::{SystemTime, UNIX_EPOCH};
-
-pub fn get_user(conn: &mut PgConnection, id: i32) -> Result<Option<User>> {
```

Comment:
This need not be removed rt?

Reply:
This was unused after removing otp based password reset and verification